src/main/resources/messages/LibertyBundles.properties

```diff
@@ -122,3 +122,16 @@ run.config.liberty.project.tool.tip=Select the build file for your Liberty proje
 run.config.start.parameters=Start Parameters
 run.config.start.parameters.tool.tip=eg. -DhotTests=true (Maven) --hotTests=true (Gradle)
 specify.custom.parameters.for.the.mvn.liberty.dev.command=Specify custom parameters for the mvn liberty:dev command
+
+# Maven and Gradle exceptions
+maven.wrapper.does.not.exist=A Maven wrapper for the project could not be found. Make sure to configure a valid Maven wrapper or change the build preferences for Maven inside IntelliJ Maven preferences.
+maven.wrapper.cannot.execute=Could not execute Maven wrapper because the process does not have permission to execute it. Consider giving executable permission for the Maven wrapper file or changing the build preferences for Maven inside IntelliJ Maven preferences.
+maven.invalid.build.preference=Make sure to configure a valid path for Maven home path inside IntelliJ Maven preferences.
+maven.cannot.execute=Could not execute Maven from {0} because the process does not have permission to execute it. Consider giving executable permission for the Maven file or configure IntelliJ to use the Maven wrapper.
+maven.does.not.exist=Could not execute the Maven file {0}. Make sure a valid path is configured inside IntelliJ Maven preferences.
+
+gradle.wrapper.does.not.exist=A Gradle wrapper for the project could not be found. Make sure to configure a valid Gradle wrapper or change the build preferences for Gradle inside IntelliJ Gradle preferences.
+gradle.wrapper.cannot.execute=Could not execute Gradle wrapper because the process does not have permission to execute it. Consider giving executable permission for the Gradle wrapper file or changing the build preferences for Gradle inside IntelliJ Gradle preferences.
+gradle.invalid.build.preference=Make sure to configure a valid path for Gradle inside IntelliJ Gradle preferences.
+gradle.cannot.execute=Could not execute Gradle from {0} because the process does not have permission to execute it. Consider giving executable permission for the Gradle file or configure IntelliJ to use the Gradle wrapper.
```

Comment:
```suggestion
gradle.cannot.execute=Could not execute Gradle from {0} because the process does not have permission to execute it. Consider giving executable permission for the Gradle executable or configure IntelliJ to use the Gradle wrapper.
```